db/migrate/20170320195445_create_books.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBooks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :books do |t|
+      t.string :title
```

Comment:
You may want to add a unique index and a uniqueness validation so you don't have duplicate books